lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1147,3 +1261,18 @@ def convert_to_utf8(input)
     end
   end
 end
+
+module Google
```

Comment:
This was only needed in the original code because we called `resource.dup`. Unless we go back to that, we shouldn't need this patch.